sql/2019/06_Fonts/06_13.sql

```diff
@@ -0,0 +1,35 @@
+#standardSQL
+
+-- counts the number of usage of FontFace API
+
+WITH
+  response_bodies AS (
+    SELECT * FROM `httparchive.response_bodies.2019_07_01_desktop`
+    UNION ALL
+    SELECT * FROM `httparchive.response_bodies.2019_07_01_mobile`
+  ),
+  temp AS (
+    SELECT
+      ARRAY_LENGTH(REGEXP_EXTRACT_ALL(body, r"new FontFace\(")) AS fontface,
```

Comment:
Let's use the blink feature counter for this metric.

```sql
SELECT
  client,
  pct_urls
FROM
  `httparchive.blink_features.usage`
WHERE
  feature = 'FontFaceConstructor' AND
  yyyymmdd = '20190701'
```

(20190701 data is unavailable until the first of next month, but you can try it with 20190601)

Reply:
changed to use blink_features